storage/store.go

```diff
@@ -30,13 +30,29 @@ type Store struct {
 	archiveReader spanstore.Reader
 }
 
+// Represents query with C-style placeholders and params for this placeholders
+type customStmt struct {
```

Comment:
Is this used only for the embedded scripts? If so I would prefer to omit this type and use`fmt.Sprintf` directly for the substitution.